Dockerfile.std

```diff
@@ -0,0 +1,21 @@
+# Generated by: tyk-ci/wf-gen
+# Generated on: Wed 17 Feb 17:51:15 UTC 2021
+
+# Generation commands:
+# ./pr.zsh -title use nfpm for std docker #2 -branch goreleaser/more -p
+# m4 -E -DxREPO=tyk
+
+FROM debian:buster-slim
```

Comment:
Could you please use `buster-20210208-slim` specifically to make sure `buster-slim` isn't cached at an earlier revision? Same for the rest of the images.